internal/backend/projects/projects.go

```diff
@@ -29,6 +29,10 @@ func New(p Projects) sdkservices.Projects { return &p }
 func (ps *Projects) Create(ctx context.Context, project sdktypes.Project) (sdktypes.ProjectID, error) {
 	project = project.WithNewID()
 
+	if !project.Name().IsValid() {
+		project = project.WithName(sdktypes.NewRandomSymbol())
```

Comment:
add random suffix just to be sure

Reply:
added random 4 digits suffix

Reply:
"How much random is random enough" rarely has a good answer. I don't like random junk in autogenerated names because it's junk, but it doesn't really matter anyway. A *robust* solution would be to regenerate or append an auto-incrementing ID if there's a duplicate, but it's not necessary for this PR.

Reply:
yeah we could be saints, but it's not needed here. i actually like adding the number suffix since it makes it even more clear that this stuff is random.